README.md

```diff
@@ -47,23 +47,21 @@ Working on the indexing pipeline effectively requires internal access to the
 Internet Archive cluster and services, though some contributions and bugfixes
 are probably possible without staff access.
 
-To install dependencies for the first time, then run the tests (to ensure
-everything is working):
```

Comment:
I think these indented lines ("make dep" and "make test") need to have empty lines before and after, so they end up as indented code blocks.

Reply:
I understand. That actually was my intent, to get them to appear indented. I will correct this instantly.
Thank you